### PR TITLE
lookupCtlTable.c: Check mallocs

### PR DESCRIPTION
**Pull request by moshekaplan**
Check a few more memory allocations

### CODE REVIEW

File: agent/mibgroup/disman/nslookup/lookupCtlTable.c [hunk 1]
```diff
@@ -137,6 +137,10 @@ create_lookupTable_data(void)
         exit(1);
     }
     StorageNew->lookupCtlTargetAddress = strdup("");
+    if (StorageNew->lookupCtlTargetAddress == NULL) {
+        free(StorageNew);
+        snmp_log(LOG_ERR, "Out in memory in nslookup-mib/create_lookupTable_date\n");
```
Comment (bvanassche) [inline agent/mibgroup/disman/nslookup/lookupCtlTable.c:142]: "Out in memory" -> "Out of memory"?

**Review by bvanassche — CHANGES_REQUESTED** [review]

File: agent/mibgroup/disman/nslookup/lookupCtlTable.c (hunk 1 above)
Comment (moshekaplan) [reply agent/mibgroup/disman/nslookup/lookupCtlTable.c]: Fixed, thanks

**Comment by moshekaplan** [comment]
@bvanassche  I'm a little confused about the other two commits that were added: Was that intentionatal?

**Comment by bvanassche** [comment]
That's probably the result of I having deleted commit "ci: Only run the Coverity action if the required secrets have been set" from the master branch. Please delete these two commits and combine the last two commits, e.g. with `git rebase -i`.

**Comment by moshekaplan** [comment]
> That's probably the result of I having deleted commit "ci: Only run the Coverity action if the required secrets have been set" from the master branch. Please delete these two commits and combine the last two commits, e.g. with `git rebase -i`.

Looks good now?

**Comment by bvanassche** [comment]
Thanks for the patch!

**Pull request merged**